test/ecpair.js

```diff
@@ -44,31 +44,6 @@ describe('ECPair', function () {
       assert.strictEqual(keyPair.network, NETWORKS.testnet)
     })
 
-    it('throws if compressed option is not a bool', function () {
-      assert.throws(function () {
-        new ECPair(null, null, {
-          compressed: 2
-        }, /Expected Boolean, got 2/)
```

Comment:
Funny enough,  this wasn't even being tested.  The regex is in the constructor call by mistake.